apps/rewards/app/store/events.js

```diff
@@ -4,7 +4,12 @@ import { addressesEqual } from '../utils/web3-utils'
 import { INITIALIZATION_TRIGGER } from './'
 
 export const handleEvent = async (state, event, settings) => {
-  const { event: eventName, returnValues, address: eventAddress, } = event
+  const {
+    event: eventName,
+    returnValues,
+    address: eventAddress,
+    transactionHash,
+  } = event
```

Comment:
Can we access the network here, rather than in the frontend? We don't want the link to change, depending on which network the user is currently connected to. We want to link them to the URL where they can view the actual transaction, whichever network that happened to be on!

Reply:
If you're on one network you won't see any rewards made on another network in the first place, so I think it's safe to access the network from the frontend.